feature/platform/tests/power_admin_down_up_test/power_admin_down_up_test.go

```diff
@@ -31,7 +31,38 @@ func TestFabricPowerAdmin(t *testing.T) {
 				t.Skipf("Fabric Component %s is already INACTIVE, hence skipping", f)
 			}
 
+			before := intfOperStatus(t, dut)
+
 			powerDownUp(t, dut, f, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_FABRIC, 3*time.Minute)
+
+			start := time.Now()
+			after := intfOperStatus(t, dut)
+			for time.Since(start) < 3*time.Minute {
+				if len(after) == len(before) {
+					break
+				}
+				time.Sleep(30 * time.Second)
+				after = intfOperStatus(t, dut)
+			}
+			if len(after) != len(before) {
+				t.Errorf("Not all Interfaces came up after Power Down Up, got: %v, want: %v", after, before)
```

Comment:
this logging statement is not helpful when the test fails, can u please change it to use https://github.com/openconfig/featureprofiles/blob/132572aa0c1c210a57642717de4bf86c57903d51/internal/helpers/helpers.go#L57  to log the exact interfaces which are not up after powerup.